pkg/common/configs/configvalidator.go

```diff
@@ -515,13 +515,12 @@ func checkLimit(limit Limit, existedUserName map[string]bool, existedGroupName m
 		return fmt.Errorf("invalid resource combination for limit %s all resource limits are null", limit.Limit)
 	}
 
-	if queue.MaxApplications != 0 {
-		if limit.MaxApplications > queue.MaxApplications {
-			return fmt.Errorf("invalid MaxApplications settings for limit %s exeecd current the queue MaxApplications", limit.Limit)
-		}
-		if limit.MaxApplications == 0 {
-			return fmt.Errorf("MaxApplications is 0 in limit name %s, it should be 1 ~ %d", limit.Limit, queue.MaxApplications)
-		}
+	if limit.MaxApplications == 0 {
+		return fmt.Errorf("MaxApplications is 0 in limit name %s, it should be 1 ~ %d", limit.Limit, queue.MaxApplications)
```

Comment:
1. MaxApplications is 0 in limit name %s -> MaxApplications is 0 in '%s' limit
2. it should be 1 ~ %d -> it should be between 1 - %d

Reply:
Updated it. Thank you.